.components/custom-layout/CustomLayout.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Banner from '../banner';
+import CoverImg from '../CoverImg';
+import Promotion from '../Promotion';
+import PromotionBannerCta from '../PromotionBannerCta';
+
+const rgxBlogRoute = /^\/blog\/?$/;
+
+const CustomLayout = (props) => {
+  const { headerComp: Header, sidepanelComp: Sidepanel, footerComp: Footer } = props;
+  const isBlogRoute = rgxBlogRoute.test(window.location.pathname);
+  
+  return (
+    <>
+      <Header />
+      <Sidepanel />
+      {!isBlogRoute && (
+        <Banner config={props.config}>
+          <div style={{ maxWidth: '768px' }}>
+            {props.frontmatter?.bannerDate || 'Date missing'}
+            <h1 id='banner-title'>{props.frontmatter?.bannerTitle || 'Title missing'}</h1>
+            {props.frontmatter?.bannerSubtitle || 'Subtitle missing'}
```

Comment:
- display the banner component only if the bannerTitle is present
- if banner subtitle or date or img isn't present then don't render the component altogether instead of having a default value like 'Title missing'